vulnfeeds/cmd/alpine/fixtures/valid_versions.txt

```diff
@@ -0,0 +1,187 @@
+
+4.0.9-r5
+2.42.2-r0
+0.9.36-r0
+5.7.1-r0
+1.0.20161017-r1
+4.11.1-r0
+1.16.0-r5
+1.7.3-r2
+2.6-r9
+1.1.1d-r1
+2.78-r0
+8.43-r1
+1.912-r0
+1.29.3-r10
+2.6-r3
+6.3_p20220416-r0
+111.1.1d-r0
+1.2.15-r11
+2.19.10.4_p8
+6.6.2p1-r0
+4.10.12-r0
+12.6-r5
+11.3-r0
+7.5_p1-r4
+4.4.3_p1-r0
+1.0-r2
+4.6.3-r10
+6.6.4p1-r2
+157.1-r2
+0.105-r9
+7.50.1
+1.1.1a-r0
+2.9-r9
+2.9-r13
+5.4-r0
+7.8-r0
+11.0.2h-r4
+20220207-r0
+6.0-r11
+2.8-r2
+0.60.6.1-r14
+1.1.1q-r0
+11.1-r4
+16.6.4p1-r1
+9.10.4_p3-r0
+12.9-r8
+6.8_p1-r7
+11.6_rc2-r5
+2.7-r5
+10.15.1b-r9
+20190618-r0
+18.38-r1
+20210608-r0
+4.4.2_p1-r0
+0.168-r1
+9.11.4_p1-r0
+16.02-r1
+2.6-r4
+2018.76-r2
+6.3_p20211120-r1
+2.9.11.6_p1-r0
+3.3.3p1-r3
+0.8_alpha-r1
+9.12.3_p4-r0
+8.3_p1-r2
+12.7-r0
+2.8-r3
+7.2_p2-r4
+7.5_p1-r2
+12.4-r0
+0.175-r0
+1.10a-r2
+4.7-r2
+5.8-r0
+1.1.24-r10
+0.28.2-r12
+4.9-r2
+7.5_p1-r0
+1.3-r1
+8.1.1365
+20220510-r0
+0.29.27-r2
+3.4-r2
+1.0.2o-r1
+2.6-r7
+0.174-r0
+6.0-r1
+11.2-r1
+118.1_p1-r1
+0.8-r4
+12.13.5
+24-r5
+6.0-r7
+19.12.1_p2-r0
+2.8.6
+2.110.42.1-r12
+2.6-r1
+2.8-r4
+19.11.1_p1-r1
+7.7_p1-r3
+11.643-r0
+55.1-r3
+0.105-r10
+7.6-r2
+7.5_p1-r3
+7.4_p1-r1
+1.0.2o-r2
+9.10.4_p8-r0
+0.60.6.1-r13
+2.4.7-r3
+10.9-r0
+15.36-r0
+7.5_p1-r10
+111.1.1n-r0
+16.0_p20170701-r0
+0
+7.8-r1
+2017.75-r0
+2022.6-r10
+13.5-r0
+2.1.33-r0
+1.0.2t-r0
+0.176-r0
+3.0a-r2
+20201112-r0
+1.8.25_p1-r3
+2019.78-r1
+14.5-r0
+0.7.1_alpha-r0
+6.7.1p1-r1
+2.7-r3
+0.15.1b-r8
+1.24.2-r13
+6.0_p20171125-r1
+1.31.1-r20
+8.3_p1-r3
+7.50.3
+25-r1
+4.6.3-r11
+1.8-r0
+1.18.0-r14
+6.0-r9
+117.7_p1-r4
+18.3_p1-r1
+1.24.2-r12
+56.1-r2
+2021.04-r0
+1.1.19-r11
+7.1.6_rc1-r0
+9.11.2_p1-r0
+2.10.18-r0
+12.9-r12
+19.10.8_p1-r0
+120191113-r0
+1.1.14-r16
+12.9-r0
+1113.8-r0
+1.0.2k-r0
+1.1.1l-r0
+2.7-r4
+2.1.26-r15
+6.2_p20210109-r1
+1.0.2h-r3
+16.8_p1-r10
+1.5.1
+2.6-r2
+12.2.9-r11
+1.1-r3
+2.9-r1
+0.91.3-r4
+12.2-r0
+0.5_alpha-r1
+7.4_p1-r2
+0.105-r8
+17.2_p1-r2
+1.0.2m-r0
+11.0.2r-r0
+13.3-r0
+16.8_p1-r8
+2.7.51.0
+110.73-r0
+1
+
```

Comment:
Do you mean to have this blank line here?

Reply:
Yep, the entries below that blank line are more borderline cases, where there is a . separating the revision part and the version, which breaks the convention set by all the other entries. Added a comment there.